_includes/configuration/gen-rsa-key-pairs.md

```diff
@@ -0,0 +1,161 @@
+
+* Red Hat Enterprise Linux 8 or later.
+* CentOS 8 or later.
+* Fedora 18 or later.
+
+```sh
+sudo dnf check-update
+sudo dnf install openssl
+```
+
+##### Debian, Ubuntu, Linux Mint, or other Debian-based distributions
+On a command line, run the following commands to install OpenSSL on Debian-based operating systems (such as: Debian, Ubuntu, and Linux Mint).
+
+```sh
+sudo apt update
+sudo apt install openssl
+```
+
+##### SUSE Linux Enterprise Server or openSUSE
+On a command line, run the following commands to install OpenSSL on openSUSE-based operating systems (such as: openSUSE and SUSE Linux Enterprise Server).
+
+```sh
+sudo zypper refresh
+sudo zypper install openssl
+```
+
+#### Create a private/public key pair on Linux
+To create a private/public key pair:
+
+1. On a command line, run:
+
+    ```sh
+    ssh-keygen -m PEM -t rsa -b 2048 -f <MY_TINY_DRIVE_KEY>
+    ```
+    Where _`<MY_TINY_DRIVE_KEY>`_ should be replaced with a name for the key pair.
+
+2. Enter a passphrase for accessing the key.
+
+Two files will be created in the current directory:
+
+* `<MY_TINY_DRIVE_KEY>` - The private key.
+* `<MY_TINY_DRIVE_KEY>.pub` - The public key.
+
+#### Retrieve the public key on Linux
+To retrieve the public key, on a command line, run:
+
+```sh
+openssl rsa -in <MY_TINY_DRIVE_KEY> -outform DER -pubout | base64 -w0
+```
+
+The public key for the `<MY_TINY_DRIVE_KEY>` key pair will be printed on the command line with base64 encoding.
+
+### Apple macOS
+
+To create a private/public key pair on a macOS operating system:
+
+1. [Create a private/public key pair](#createaprivatepublickeypaironmacos).
+1. [Retrieve the public key](#retrievethepublickeyonmacos).
+
+#### Create a private/public key pair on macOS
+To create a private/public key pair:
+
+1. Using **Finder**, open a **Terminal**.
+1. On a terminal, run:
+
+    ```sh
+    ssh-keygen -m PEM -t rsa -b 2048 -f <MY_TINY_DRIVE_KEY>
+    ```
+    Where _`<MY_TINY_DRIVE_KEY>`_ should be replaced with a name for the key pair.
+
+1. Enter a passphrase for accessing the key.
+
+Two files will be created in the current directory:
+
+* `<MY_TINY_DRIVE_KEY>` - The private key.
+* `<MY_TINY_DRIVE_KEY>.pub` - The public key.
+
+#### Retrieve the public key on macOS
+To retrieve the public key, on a terminal, run:
+
+```sh
+openssl rsa -in <MY_TINY_DRIVE_KEY> -outform DER -pubout | base64 -
+```
+
+The public key for the `<MY_TINY_DRIVE_KEY>` key pair will be printed on the terminal with base64 encoding.
+
+### Microsoft Windows
+To create a private/public key pair on a Microsoft Windows operating system:
+
+1. [Install OpenSSL](#installingopensslonmicrosoftwindows).
+1. [Create a private/public key pair](#createaprivatepublickeypaironwindows).
+1. [Retrieve the public key](#retrievethepublickeyonwindows).
+
+#### Installing OpenSSL on Microsoft Windows
+To install OpenSSL with _Git for Windows_:
+
+1. Download the _Windows_ package from [git Downloads](https://git-scm.com/downloads).
```

Comment:
Should this be a capital `D` for `Downloads`? It just looks a bit weird is all, but maybe that's just me :)

Reply:
it looks weird, but it's correct. I've updated it to "the git Downloads page" to make it look less weird?? at least it's clearer.